src/search/preview/CallToAction.tsx

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable lingui/no-unlocalized-strings */
+import { Box, Button, Link, Typography } from "@mui/material";
+
+interface Props {
+  domain: string;
+}
+
+export function CallToAction({ domain }: Props) {
+  return (
+    <Box
+      marginBottom={1}
+      padding={2}
+      display="flex"
+      gap={1}
+      alignItems="center"
+      bgcolor="#F7F9FF"
+      borderRadius="12px"
+    >
+      <Box flex={1}>
+        <Typography variant="h2" marginBottom={2}>
+          Brave Search Ads Preview
+        </Typography>
+        <Typography variant="body2" marginBottom={2}>
+          This preview has been created for{" "}
+          <Link
+            href={`https://${domain}/`}
+            target="_blank"
+            rel="noreferrer"
+            underline="hover"
```

Comment:
```suggestion
            underline="hover"
            variant="inherit"
```

Reply:
this is the default - https://mui.com/material-ui/api/link/